src/SW_Files.c

```diff
@@ -362,7 +360,7 @@ void SW_F_construct(
             (void *) SW_ProjDir, (void *) c, '\0', sizeof dirString
         );
         if (isnull(resPtr)) {
-            sw_memccpy_report(swFALSE, swTRUE, endProjDir, LogInfo);
+            reportFullBuffer(LOGWARN, LogInfo);

```

Comment:
SW_F_construct() -- same comment as for SW_CSV_F_INIT(): should we replace sw_memccpy() with sw_memccpy_inc() to set the last byte to null on full buffer?
Also, it seems safer to fail if there is a full buffer (instead of issuing a warning) - to make clear that something is wrong with output files.